### PR TITLE
docs(v2): clarify theme-original and theme-init

### PR DESCRIPTION
**Pull request by Jonathannsegal**
## Motivation

https://github.com/facebook/docusaurus/issues/3043

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

Change to `api-pages.md` Page Require Paths

## Related PRs

As explained by https://github.com/facebook/docusaurus/pull/2464#issuecomment-603802459

### CODE REVIEW

**Comment by slorber** [comment]
Hi and thanks.

Unfortunately, after reading that doc I'm still not 100% convinced to understand those aliases and why they are useful in practice 🤪

Will need to understand first, and see how we could improve the doc later

File: docs/api-pages.md [hunk 1]
```diff
@@ -83,6 +83,8 @@ What this means to the user is that if you wish to use the `CompLibrary` module,
 
 If you wish to use your own components inside the website directory, use `process.cwd()` which will refer to the `website` directory to construct require paths. For example, if you add a component to `website/core/mycomponent.js`, you can use the require path, `'process.cwd() + /core/mycomponent.js'`.
 
+There is a special import for custom items `@theme-original`. The `theme-original` alias (just like using `theme` alias) will not get the theme component from the plusin's code. While the `init-theme` alias refers to the proper (theme) component (from the theme itself, where it is first defined). Therefore the `theme-original` is for the user and `theme-initial` is for the plugins.
```
Comment (Drewbi) [inline docs/api-pages.md:86]: Typo
```suggestion
There is a special import for custom items `@theme-original`. The `theme-original` alias (just like using `theme` alias) will not get the theme component from the plugin's code. While the `init-theme` alias refers to the proper (theme) component (from the theme itself, where it is first defined). Therefore the `theme-original` is for the user and `theme-initial` is for the plugins.
```

**Comment by slorber** [comment]
Let's merge this for now, but keep the issue open, because I still feel the doc can be clearer

**Comment by slorber** [comment]
Oh, that's unfortunate but this has been contributed to v1 docs while it's a v2 feature :( will fix that soon

**Pull request merged**